File: src/main/java/com/dh/ondot/member/core/AppleProperties.java
package com.dh.ondot.member.core;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "oauth2.client.registration.apple")
public record AppleProperties(
        String clientId,
        String teamId,
        String keyId,
        String audience,
        String grantType,
        String privateKey
) {
}

File: src/main/kotlin/com/dh/ondot/member/core/AppleProperties.kt
package com.dh.ondot.member.core

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "oauth2.client.registration.apple")
data class AppleProperties(
    val clientId: String,
    val teamId: String,
    val keyId: String,
    val audience: String,
    val grantType: String,
)
